Content.Client/SubFloor/TrayScannerSystem.cs

```diff
@@ -61,6 +62,15 @@ public override void Update(float frameTime)
                     canSee = true;
                     range = MathF.Max(range, sneakScanner.Range);
                 }
+
+                foreach (var ent in slot.ContainedEntities)
+                {
+                    if (!wearableScannerQuery.TryGetComponent(ent, out var sneakScanner) || !sneakScanner.Enabled)
+                        continue;
+
+                    canSee = true;
+                    range = MathF.Max(range, sneakScanner.Range);
+                }
```

Comment:
slot's review its still valid, this code duplication is not really desirable and i dont think its that necessary? if you dont want people to activate the goggles in-hand, get rid of the new component and have them work then just add a nullable (either `string` or `SlotFlags`) AllowedSlots datafield to tray component and make the necessary checks in the current slots and hand enumerators

Reply:
I really just dont know how to do that so im probably just going to close this pr if i need to do this.